modules/imaging_browser/php/imaging_session_controlpanel.class.inc

```diff
@@ -83,6 +83,7 @@ class Imaging_Session_ControlPanel
                  'v_sessions_id' => $this->sessionID,
                 )
             );
+            print_r($qresult);
```

Comment:
Hehe, looks like you left a print here :)

![screen shot 2018-11-30 at 10 45 56 am](https://user-images.githubusercontent.com/1402456/49299244-46c9da80-f48d-11e8-822d-b65c28688684.png)